src/it/scala/org/ergoplatform/it/FourNodeSyncSpec.scala

```diff
@@ -22,12 +22,12 @@ class FourNodeSyncSpec extends FreeSpec with IntegrationSuite {
 
   s"Generate $blocksCount blocks" in {
     val result = for {
-      b <- Future.traverse(nodes)(_.height).map(_.max)
-      _ <- Future.traverse(nodes)(_.waitForHeight(b + blocksCount))
-      headers <- Future.traverse(nodes)(_.headerIdsByHeight(b + blocksCount - forkDepth))
+      maxInitialHeight <- Future.traverse(nodes)(_.height).map(_.max)
+      reachedHeight <- Future.traverse(nodes)(_.waitForHeight(maxInitialHeight + blocksCount)).map(_.max)
+      headers <- Future.traverse(nodes)(_.headerIdsByHeight(reachedHeight + blocksCount - forkDepth))
     } yield {
-      log.debug(s"Headers at height ${b + blocksCount - forkDepth}: ${headers.mkString(",")}")
-      val headerIdsAtSameHeight = headers.flatten
+      log.debug(s"Headers at height ${reachedHeight + blocksCount - forkDepth}: ${headers.mkString(",")}")
+      val headerIdsAtSameHeight = headers.map(_.head)
```

Comment:
@mike-aksarin Could you please check that I did not lose the whole test idea here. Is it enough to make sure the best blocks are the same, or we need to be strict and require no forking(no multiple headers)?